documentation/library_guide/parallel_api/pass_data_algorithms.rst

```diff
@@ -30,6 +32,36 @@ with parallel execution policies.
 
 The following subsections describe proper ways to pass data to an algorithm invoked with a device execution policy.
 
+.. _indirectly-device-accessible:
+
+Indirectly Device Accessible
+----------------------------
+Iterators and iterator-like types may or may not refer to content accessible within a
+`SYCL <https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html>`_ kernel on a device. The term
+*indirectly device accessible* refers to a type that represents content accessible on a device. An indirectly device
+accessible iterator is a type that can also be dereferenced within a SYCL kernel.
```

Comment:
Do I get it right that in order to be considered indirectly device copyable the type must provide `true` when with `is_onedpl_indirectly_device_accessible_v`?

If so, Let's unite that section with "Indirect Device Accessibility Type Trait".

Reply:
I think the term *indirectly device accessible* is more a general thing representing the semantic meaning described here. The trait allows oneDPL to properly treat types which are *indirectly device accessible*.  That being said, I dont have an objection to combining the sections.

Reply:
combined